website/views/emails/email-reset-password.ejs

```diff
@@ -3,6 +3,6 @@
 <p style="margin-bottom: 32px;">Someone requested a password reset for your Fleet account.<br>Please click below to reset your password:</p>
 
   <div style="background: #6A67FE; display: inline-block; color: white; text-decoration: none; border-radius: 4px; text-align: center; width: 151px; padding: 8px 16px 8px 16px" >
-    <a style="margin: auto; font-size: 16px; text-decoration: none; color: white; line-height: 22px;" href="<%= url.resolve(sails.config.custom.baseUrl,'/customers/new-password')+'?token='+encodeURIComponent(token) %>">Update password</a>
```

Comment:
I agree with this change for the sake of readability, but since this is just adding a space character, I'd omit this change from this PR to avoid adding this to the `git blame` and making us think emails were changed in this PR (just since there's been a lot of back and forth and we'll probably kinda forget)